cli/cli.py

```diff
@@ -779,12 +803,14 @@ def desbordante_cli(**kwargs: Any) -> None:
     curr_algo = ALGOS[curr_algo_name]()
     error_opt = kwargs[ERROR]
     error_measure_opt = kwargs[ERROR_MEASURE]
+    input_format_opt = kwargs[INPUT_FORMAT]
     verbose = kwargs[VERBOSE]
     filename = kwargs[FILENAME]
 
     check_mismatch(curr_algo_name, curr_task)
     check_error_option_presence(curr_task, error_opt)
     check_error_measure_option_presence(curr_task, error_measure_opt)
+    check_input_format_option_presence(curr_task, input_format_opt)
```

Comment:
Isn't the algorithm complaining about this option not being present enough?

Reply:
I little bit dont understand what about talking. If it is needed, then yes, the algorithm cannot work without this option.

Reply:
I'm saying that because the algorithm itself errors out if one of the options without a default value, which I presume 'input_format' is, is not set, there shouldn't be a need for an additional check in the CLI.